content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb

```diff
@@ -0,0 +1,148 @@
+---
+title: Review and streamline how we communicate
+---
+
+<div class="govuk-main-wrapper">
+  <div class="dfe-width-container govuk-grid-row two-column-page">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="dfe-width-container govuk-grid-row">
+        <h1 class="govuk-heading-l">Review and streamline how we communicate</h1>
+        <div class="dfe-width-container govuk-grid-row govuk-!-margin-bottom-9">
+          <strong class="govuk-tag">Presentation</strong>
+        </div>
+        <div class="dfe-width-container govuk-grid-row">
+          <p>
+            Use this presentation in a staff meeting or INSET day to review and streamline your internal and external
+            communication processes.
+          </p>
+          <p>
+            The presentation includes:
+          <ul>
+            <li>background and context about managing communications</li>
+            <li>a discussion to reflect on why we communicate and whether the processes should be updated, streamlined
```

Comment:
could we separate this into 2 bullet points please:
<li>a reflection on why we communicate</li>
<li>a discussion on whether current communication processes should be updated, streamlined or stopped altogether</li>